src/app/[id]/page.tsx

```diff
@@ -932,579 +956,853 @@ homeless men, etc.) This helps us to make appropriate referrals."
     return service_items;
   };
 
+  const convertToArray = (input: string) => {
+    if (Array.isArray(input)) return input;
+    else if (input == '') return [];
+    else if (input.includes(',')) {
+      const str = input.replace(/\s/g, '');
+      return str.split(',');
+    } else return [input];
+  };
+
+  interface preliminaries_header {
+    name: string;
+  }
+
+  const PreliminariesHeader = (data: preliminaries_header) => {
```

Comment:
just a small suggestion, but I would probably remove the interface and just add data: {name: string}